lib/kubeclient.rb

```diff
@@ -53,6 +53,7 @@ class Client
 
     DEFAULT_HTTP_PROXY_URI = nil
     DEFAULT_HTTP_MAX_REDIRECTS = 10
+    DEFAULT_KEEP_ALIVE_TIMEOUT = 60
```

Comment:
The default of `http` gem seems to be 5 seconds: https://github.com/httprb/http/blob/v4.4.1/lib/http/options.rb#L63

I open to setting it higher by default if that's good for k8s watch connections.
This effectively controls how long we keep the watch open when server has no updates, right?
Do you know what's the default in Go or other k8s clients?

OTOH if we're not sure what's best value, we can set it 5 here to keep existing behavior, this PR will let users experiment...